.gitlab-ci.yml

```diff
@@ -64,8 +64,9 @@ ci-authorized:
   only: []
 
 include:
-  - .gitlab-ci/lint.yml
-  - .gitlab-ci/shellcheck.yml
-  - .gitlab-ci/digital-ocean.yml
-  - .gitlab-ci/terraform.yml
-  - .gitlab-ci/packet.yml
+  - .gitlab-ci/molecule.yml
+  # - .gitlab-ci/lint.yml
```

Comment:
Note to reviewers: this (and following 4 lines) have to be removed before merging

Reply:
Thank you for highlighting that. I edited the PR description and added information that the two tip commits are for testing purposes only. They should not be merged.